packages/server/graphql/mutations/addPokerTemplateScaleValue.ts

```diff
@@ -0,0 +1,92 @@
+import {GraphQLID, GraphQLInt, GraphQLNonNull} from 'graphql'
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+import getRethink from '../../database/rethinkDriver'
+import {getUserId, isTeamMember} from '../../utils/authorization'
+import publish from '../../utils/publish'
+import standardError from '../../utils/standardError'
+import TemplateScaleInput from '../types/TemplateScaleInput'
+import AddPokerTemplateScaleValuePayload from '../types/AddPokerTemplateScaleValuePayload'
+import {
+  validateColorValue,
+  validateScaleLabel,
+  validateScaleValue
+} from './helpers/validateScaleValue'
+
+const addPokerTemplateScaleValue = {
+  description: 'Add a new scale value for a scale in a poker template',
+  type: AddPokerTemplateScaleValuePayload,
+  args: {
+    scaleId: {
+      type: new GraphQLNonNull(GraphQLID)
+    },
+    scaleValue: {
+      type: new GraphQLNonNull(TemplateScaleInput)
+    },
+    index: {
+      type: GraphQLInt
+    }
+  },
+  async resolve(
+    _source,
+    {scaleId, scaleValue, index},
+    {authToken, dataLoader, socketId: mutatorId}
+  ) {
+    const r = await getRethink()
+    const now = new Date()
+    const operationId = dataLoader.share()
+    const subOptions = {operationId, mutatorId}
+    const viewerId = getUserId(authToken)
+
+    // AUTH
+    const scale = await r
+      .table('TemplateScale')
+      .get(scaleId)
+      .run()
+    if (!scale || scale.removedAt) {
+      return standardError(new Error('Did not find an active scale'), {userId: viewerId})
+    }
+    if (!isTeamMember(authToken, scale.teamId)) {
+      return standardError(new Error('Team not found'), {userId: viewerId})
+    }
+
+    // VALIDATION
+    const endIndex = scale.values.length - 1
+    if (index > endIndex + 1 || index < 0) {
+      return standardError(new Error('Invalid index'), {userId: viewerId})
+    }
+    const {color, label, value, isSpecial} = scaleValue
+    if (isSpecial) {
```

Comment:
we may allow users to remove the ? card.
if that's the case, we should allow them to add it back in.
therefore, we should allow `isSpecial`, but make sure that the value is -1 or 2**31-1